### DISCUSSION TITLE
Exclude in the home page - Releases/Packages/Environments - doesn't work

### DISCUSSION THREAD

**Discussion by mkosir** [community/community#62235] [Repositories]
### Select Topic Area

Bug

### Body

Trying to turn of the Releases/Packages/Environments from showing them on repository, but doesn't seem to work.

https://github.com/community/community/assets/3073826/ae90d8b6-2328-4c84-9079-681621e48cc1

**Comment by StevenEWright** [comment]
Dupe of: https://github.com/orgs/community/discussions/62025
